styles/app/components/sidebar/sidebar.module.css

```
@@ -6,21 +6,14 @@
 	max-width: 315px;
 	width: 100%;
 	height: calc(100vh - 2rem);
-	padding: 20px;
 	background: var(--color-background);
 	border: 2px solid var(--switch-border);
 	border-radius: 10px;
 
 	header {
-		padding-bottom: 15px;
+		padding: 20px 20px 15px 20px;
```

Tu peux le remplacer par ça

```suggestion
		padding: 20px 20px 15px;
```

Cela correspond à `padding: left y right`